src/components/overlay/theme.css

```diff
@@ -13,6 +13,6 @@
 
 .is-entered {
   &.dark {
-    background-color: color(var(--color-teal-dark) a(85%));
+    background-color: color(var(--color-teal-dark) a(50%));
```

Comment:
Is this decided by a designer? I don't see this change in figma.

Reply:
Yup, @driesd told me to implement this, and he was told by our resident designer Stephen 🙂